src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -23,8 +23,8 @@
     private static final int BIG_LABEL_FONT = 16;
 
     private boolean isSelected = false;
-    private boolean isMatching = false;
     private boolean isExisting = false;
+    private boolean isMatching = true;
```

Comment:
Just wondering if it makes a difference for `isMatching` here to be `true` or `false`.